config/http.conf

```diff
@@ -3,7 +3,7 @@ http {
   include /etc/nginx/log.conf;
 
   server_tokens         off;
-  keepalive_timeout     20s;
+  keepalive_timeout     450s;
```

Comment:
Because LBs are by default 60s so we need higher than that. Whoever's using this image will have to set their application time out to be higher than 450s. To note, the right value for this is still being discussed (will update once concluded). Personally, I'm hoping we can set this to 75 just like the [Kubernetes Nginx Ingress](https://kubernetes.github.io/ingress-nginx) ([see here](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#upstream-keepalive-timeout)).